docs/doxygen/generate_docs.bash

```diff
@@ -75,6 +75,7 @@ function clobber
 # Python
 (
     clobber "${PY_OUTPUT}"
+    pip install sphinx sphinx-rtd-theme sphinx-autoapi sphinx-autoapi recommonmark sphinxcontrib.mermaid
```

Comment:
Why is this line needed?  It seems like it should already be part of the VENV.

Reply:
I agree, they could be moved to 
https://github.com/nasa/fprime/blob/261ae546d93feb779536bee393caeb8c5140167d/Fw/Python/setup.py#L87

extras_require={"dev": ["black", "pylama", "pylint", "pre-commit"]},